backend/frontend/src/components/GoogleMapContainer.jsx

```diff
@@ -197,7 +247,54 @@ export const GoogleMapContainer = (props) => {
                     <div className='Review'> {parse(reviewlist)} </div>
                     <br></br><br></br>
 
-                    <button className="buttonaddreview">Add Review</button>
+                    <Button variant="primary" onClick={handleShowModal}>Add Review</Button>
+
+                    <Modal show={showModal} onHide={handleCloseModal}>
+                        <Modal.Header closeButton>
+                            <Modal.Title>New Review</Modal.Title>
+                        </Modal.Header>
+                        <Modal.Body>
+                            <form onSubmit={handleSubmit}>
+                            <label>{amenity_type} {amenity_id} </label>
+                            <label>Rating (1-5):
+                            <input 
+                                type="number" 
+                                name="rating" 
+                                value={inputs.rating || ""}
+                                required = "required" 
+                                onChange={handleChange}
+                                min={1}
+                                max={5}
+                            />
+                            </label>
+                            <label>Review:
+                                <input 
+                                    type="text" 
+                                    name="review" 
+                                    value={inputs.review || ""} 
+                                    required = "required" 
```

Comment:
lgtm. 
Is the review "required" though?

Reply:
Agreed. Lets let it pass for now and revisit in a later sprint